storage/src/checker/range_set.rs

```diff
@@ -269,23 +271,38 @@ impl LinearAddressRangeSet {
                 intersection,
             });
         }
+        self.bytes_in_set = self
+            .bytes_in_set
+            .checked_add(size)
+            .expect("Overflow is not possible");
         Ok(())
     }
 
     pub(super) fn complement(&self) -> Self {
         let complement_set = self
             .range_set
             .complement(&Self::NODE_STORE_START_ADDR, &self.max_addr);
-
+        let bytes_in_complement = self
+            .max_addr
+            .get()
+            .checked_sub(NodeStoreHeader::SIZE)
+            .expect("Impossible due to check in new()")
+            .checked_sub(self.bytes_in_set)
+            .expect("Underflow is not possible since total_bytes only gets incremented when inserting disjoint area within [NODE_STORE_ADDR_START, max_addr)");
```

Comment:
Seems like some helpers on LinearAddress would do wonders here. Why not mplement checked_sub on LinearAddress?

nit: wordy expects
try:
"checked in new()" and "bytes_in_set is always less than max_addr"